provider/resources.go

```diff
@@ -90,10 +92,10 @@ func Provider() tfbridge.ProviderInfo {
 			"spotinst_managed_instance_aws":       {Tok: makeResource(awsMod, "ManagedInstance")},
 			"spotinst_elastigroup_azure":          {Tok: makeResource(azureMod, "Elastigroup")},
 			"spotinst_ocean_aks":                  {Tok: makeResource(azureMod, "Ocean")},
-			"spotinst_ocean_aks_np": {
-				Tok:  makeResource(azureMod, "OceanNp"),
-				Docs: noUpstreamDocs,
-			},
+			// "spotinst_ocean_aks_np": {
+			// 	Tok:  makeResource(azureMod, "OceanNp"),
+			// 	Docs: noUpstreamDocs,
+			// },
```

Comment:
why is this commented out as opposed to removed?

Reply:
No good reason honestly, thought it might be useful to show that the resources were removed in recent upgrade

Reply:
Let's remove them then.